aruco_msgs/msg/MarkerImage.msg

```diff
@@ -1,2 +1,3 @@
 Marker[] markers
 sensor_msgs/Image image
+DetectionZone detection_zone
```

Comment:
This will affect big too, correct?

Reply:
Yes, though I've kept the default detection zone to be the entire image. This will need to sync up with https://github.com/braincorp/shining_software/pull/3917 and https://github.com/braincorp/emma_ui/pull/210 though.

Reply:
If you remove the rotate/crop/scale functions from here, do you also need to add rotate/crop/scale to big's version of the UI (in `assets/big_1.0/qml/HomingScreen.qml`)?

Reply:
No, fortunately they only use scaling at the QML level and don't need any of these features in aruco_ros.